realm/realm-library/src/objectServer/java/io/realm/SyncUser.java

```diff
@@ -462,6 +477,15 @@ private static String getManagementRealmUrl(URL authUrl) {
         }
     }
 
+    /**
+     * FIXME Javadoc
+     *
+     * @return
+     */
+    public PermissionManager getPermissionManager() {
```

Comment:
Ideally, the constructor wont be public. Trying to figure out a way to make it not so. The PermissionManager should be closed just Realms. I'll expand the javadoc description.